src/current/v22.2/scalar-expressions.md

```diff
@@ -397,14 +397,15 @@ SQL supports function overloading. See [Revisiting SQL Typing in CockroachDB](ht
 
 ## Subscripted expressions
 
-It is possible to access one item in an [`ARRAY`](array.html) or [`JSONB`](jsonb.html) value using the `[` ... `]` operator. For example:
+You can use the `[` ... `]` operator to access one item in an [`ARRAY`]({% link {{ page.version.version }}/array.md %}) or [`JSONB`]({% link {{ page.version.version }}/jsonb.md %}) value. For example:
 
-- If the name `a` refers to an array of 10 values, `a[3]` will retrieve the 3rd value. The first value has index 1.
+- If the name `a` refers to an `ARRAY` of 10 values, `a[3]` will retrieve the third value. The first value in an `ARRAY` has index `1`.
+- If the name `j` refers to a `JSONB` array of 10 values, `j[3]` will retrieve the fourth value. The first value in a `JSONB` array has index `0`.
```

Comment:
The commonly used terms here are `zero-based indexing` and `1-based indexing`, if you want to incorporate those for clarity.

Reply:
TFTR! Will add this.